astro/src/content/quickstarts/quickstart-php-laravel-api.mdx

```diff
@@ -0,0 +1,424 @@
+* Your example customer username is <code><RemoteValue url={frontmatter.coderoot + "/kickstart/kickstart.json"} selector="$.variables.customerEmail" /></code> and the password is <code><RemoteValue url={frontmatter.coderoot + "/kickstart/kickstart.json"} selector="$.variables.customerPassword" /></code>. They will have the role of `customer`.
+* Your admin username is <code><RemoteValue url={frontmatter.coderoot + "/kickstart/kickstart.json"} selector="$.variables.adminEmail" /></code> and the password is <code><RemoteValue url={frontmatter.coderoot + "/kickstart/kickstart.json"} selector="$.variables.adminPassword" /></code>.
+* The base URL of FusionAuth is `http://localhost:9011/`.
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+
+### Expose FusionAuth Instance
+
+To make sure your local FusionAuth instance is accessible to your Laravel app, you need to [expose it to the Internet](/docs/v1/tech/developer-guide/exposing-instance). Write down the URL ngrok gave you as you'll need it soon.
+
+## Create your Laravel Application
+
+Now you are going to create a Laravel API application. While this section builds a simple API, you can use the same configuration to integrate an existing API with FusionAuth.
+
+We are going to be building an API backend for a banking application called ChangeBank. This API will have two endpoints:
+- `make-change`: This endpoint will allow you to call GET with a `total` amount and receive a response indicating how many nickels and pennies are needed to make change. Valid roles are `customer` and `teller`.
+- `panic`: Tellers may call this endpoint to call the police in case of an incident. The only valid role is `teller`.
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in the `app.at` cookie in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+<Aside type="note">
+  If you simply want to run the application, there is a completed version in the `complete-application` directory. You can install [PHP](https://www.php.net) and [Composer](https://getcomposer.org/) and use the following commands to get it up and running if you do not want to create your own.
+
+  ```shell
+  cd complete-application
+  composer install
+  ./vendor/bin/sail up -d
+  ```
+
+  You can then follow the instructions in the "Run the API" section.
+</Aside>
+
+To create the application from scratch, you'll use [Laravel Sail](https://laravel.com/docs/10.x/installation#laravel-and-docker) to set up and run your application using Docker.
+
+```shell
+curl "https://laravel.build/fusionauth-quickstart-php-laravel-api?with=mariadb" | bash
+cd fusionauth-quickstart-php-laravel-api
+```
+
+### Initialize the Application
+
+Initialize the Laravel application using the following:
+
+```shell
+./vendor/bin/sail up
```

Comment:
Does this just run in the terminal window?  I was waiting for it to finish for a long time.  Should add a note that this need to be running in the window and to open a new terminal window for next commands.